src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -132,19 +132,16 @@
               (switcher-cards-store/fetch-switcher-cards-rpc))))
 
 (rf/defn login-node-signal
-  [{{:keys [recovered-account?] :as db} :db :as cofx} {:keys [settings account error]}]
+  [{{:onboarding-2/keys [recovered-account? new-account?] :as db} :db :as cofx} {:keys [settings account error]}]
   (log/debug "[signals] node.login" "error" error)
   (if error
     {:db (update db :profile/login #(-> % (dissoc :processing) (assoc :error error)))}
-    (let [{:keys [creating?]} (:profile/login db)]
-      (rf/merge cofx
-                {:db         (dissoc db :profile/login)
-                 :dispatch-n [[:logging/initialize-web3-client-version]
-                              (when (and creating? (not recovered-account?))
-                                [:wallet/set-initial-blocks-range])]}
-                (if (or creating? recovered-account?)
-                  (profile.create/login-new-profile recovered-account?)
```

Comment:
It looks like `(profile.create/login-new-profile recovered-account?)` was dead as both `creating?` and `recovered-account?` would be always `nil` nowadays, so I just removed it.